### PR TITLE
New package: wayout-0.1.5

### PR DESCRIPTION
**Pull request by elbachir-one**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (ARCH-LIBC)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv6l-musl
  - aarch64-musl
  - armv7l

Addresses: #51491

### CODE REVIEW

File: srcpkgs/wayout/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'wayout'
+pkgname=wayout
+version=0.1.5
+revision=1
+build_style=meson
+hostmakedepends="meson ninja scdoc pkg-config cmake"
+makedepends="wayland wayland-protocols pango-devel wayland-devel cairo-devel"
```
Comment (abenson) [inline srcpkgs/wayout/template:7]:
```suggestion
makedepends="wayland-protocols pango-devel wayland-devel cairo-devel"
```

File: srcpkgs/wayout/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+depends="wayland cairo pango wayland-protocols"
```
Comment (abenson) [inline srcpkgs/wayout/template:8]:
```suggestion
```

File: srcpkgs/wayout/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'wayout'
+pkgname=wayout
+version=0.1.5
+revision=1
+build_style=meson
+hostmakedepends="meson ninja scdoc pkg-config cmake"
```
Comment (abenson) [inline srcpkgs/wayout/template:6]:
```suggestion
hostmakedepends="meson ninja scdoc pkg-config cmake wayland-devel"
```

Comment (abenson) [reply srcpkgs/wayout/template:6]: This will fix the crossbuild.

File: srcpkgs/wayout/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/wayout/template:7]: Don't need `wayland` in the `makedepends`, just the `-devel`.

File: srcpkgs/wayout/template (hunk 2 above)
Comment (abenson) [reply srcpkgs/wayout/template:8]: These will be automatically detected.

**Comment by abenson** [comment]
You will need to [rebase and squash](https://www.git-tower.com/learn/git/faq/git-squash) these commits together. Once you do that, you can force push (`git push -f`) to this branch.

**Comment by elbachir-one** [comment]
Sorry about this mess.

**Pull request closed without merging**